runtime/src/main/java/io/quarkus/oidc/proxy/runtime/OidcProxy.java

```diff
@@ -55,6 +61,7 @@ public OidcProxy(TenantConfigBean tenantConfig, OidcProxyConfig oidcProxyConfig,
         this.oidcProxyConfig = oidcProxyConfig;
         this.configuredClientSecret = OidcCommonUtils.clientSecret(oidcTenantConfig.credentials);
         this.httpRootPath = httpRootPath;
+        this.tokenEncryptionKey = tenantConfigContext.getTokenDecryptionKey();
```

Comment:
This looks strange, mind adding a commebt explaining why?

Reply:
@gastaldi Sure, let me clean up it a bit, that works with the symmetric secret key, but indeed, it won't make sense for the asymmetric case.